ui/media/js/image-modifiers.js

```diff
@@ -283,4 +283,13 @@ function loadCustomModifiers() {
     }
 }
 
+// Empty the modifiers when reusing existing settings to avoid duplicate modifiers.
+// Restoring the modifiers will require more in-depth changes. This is a quick workaround for now.
+function clearModifiers() {
+	activeTags.forEach(tag => {
+		tag.originElement.classList.remove(activeCardClass)
+	})
+	activeTags.length = 0
```

Comment:
How about `activeTags = []`? `.length` could behave oddly: https://www.tutorialspoint.com/in-javascript-how-to-empty-an-array

Reply:
I removed this code and now have proper support for restoring the Modifiers in the left pane.